bin/genparser

```diff
@@ -236,6 +236,7 @@ do
     echo '#pragma GCC diagnostic ignored "-Wsign-conversion"                ' >> tmp.out
     echo '#pragma GCC diagnostic ignored "-Wstrict-overflow"                ' >> tmp.out
     echo '#pragma GCC diagnostic ignored "-Wstrict-prototypes"              ' >> tmp.out
+    echo '#pragma GCC diagnostic ignored "-Wimplicit-fallthrough"           ' >> tmp.out
```

Comment:
I didn't re-run genparser (yet). I just moved the pragma over in the appropriate files.